### PR TITLE
docs: Add logo of bash-it to readme

### PR DESCRIPTION
**Pull request by NoahGorny**
Add the logo of Bash-it to the readme- makes the readme more pretty and engaging!

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-# Bash-it
+![docker](https://github.com/Bash-it/media/raw/master/media/Bash-it.png)
```
Comment (cornfeedhobo) [inline docs/README.md:1]: You've made the alt text `docker`, and I think you actually want `logo`.

Comment (NoahGorny) [reply docs/README.md]: whoops, nice catch!

**Comment by NoahGorny** [comment]
@cornfeedhobo I added some more badges, to make the readme prettier :smile:

**Review by cornfeedhobo — APPROVED** [review]
Thanks!

**Pull request merged**